alma/controllers/front/payment.php

```diff
@@ -28,11 +28,17 @@
 use Alma\API\RequestError;
 use Alma\PrestaShop\API\ClientHelper;
 use Alma\PrestaShop\Model\PaymentData;
+use Alma\PrestaShop\Utils\AjaxTrait;
 use Alma\PrestaShop\Utils\Logger;
+use Alma\PrestaShop\Utils\OrderDataTrait;
 use Alma\PrestaShop\Utils\Settings;
 
 class AlmaPaymentModuleFrontController extends ModuleFrontController
 {
+    use AjaxTrait, OrderDataTrait {
+        AjaxTrait::ajaxFail insteadof OrderDataTrait;
+    }
+
     public $ssl = true;
```

Comment:
unused variable

Reply:
@ClaireAlma 
is used here `$this->ajaxFail();` line 76, 83, 98, 106...